customizations.md

```diff
@@ -30,7 +33,7 @@ Finally, in order to install a delta image, an ECU SHOULD take one of the action
 
 ### TODO Recreate and insert Table D.2a from google doc and label it as Table 1.
 
-
+                                                                                                                                                                                                                 |
```

Comment:
This weird little extra change seems superfluous.

Reply:
Fixed this